src/site/content/en/payments/registering-a-web-based-payment-app/index.md

```diff
@@ -0,0 +1,321 @@
+---
+layout: post
+title: Registering a web-based payment app
+subhead: |
+  Learn how to configure a web-based payment app on registration.
+authors:
+  - agektmr
+date: 2020-07-17
+description: |
+  Learn how to register a web-based payment app to a customers' browser. You'll also learn how to debug them.
+tags:
+  - payments
+---
+
+Web-based payment apps are [Progressive Web Apps
+(PWA)](/progressive-web-apps/) and run on top of [service
+workers](https://developer.mozilla.org/docs/Web/API/Service_Worker_API). The
+service worker in a payment app plays an important role as it captures payment
+requests from a merchant, launches the payment app, and mediates the
+communication with the merchant.
+
+To configure a web-based payment app, you need to register available payment
+methods, delegation options, and a service worker. You can configure your
+web-based payment app declaratively with a web app manifest or imperatively with
+JavaScript.
+
+## Configuring a payment app with a web app manifest
+
+To configure your web-based payment app declaratively, [serve a web app
+manifest](/setting-up-a-payment-method/#step-3:-serve-a-web-app-manifest).
+
+The following properties in the web app manifest are relevant for web-based payment apps:
+* `name`
+* `icons`
+* `serviceworker`
+    * `src`
+    * `scope`
+    * `use_cache`
+* `payment`
+    * `supported_delegations`
+
+Check out [Setting up a payment
+method](/setting-up-a-payment-method/#step-3:-serve-a-web-app-manifest)
+to make sure your payment method manifest points to your web app manifest
+properly.
+
+### Registering a service worker just-in-time (JIT)
+
+Service workers are usually registered using JavaScript, but they can also be
+automatically registered by the browser when the user chooses to pay with a
+web-based payment app on a merchant website. This is called "just-in-time (JIT)
+registration".
+
+The JIT registration requires only serving [the web app
+manifest](/setting-up-a-payment-method/#step-3:-serve-a-web-app-manifest)
+and no additional coding. If you've already configured your web app manifest and
+are serving it properly, you should be all set. The browser will handle the
+rest.
```

Comment:
All browsers support JIT registration?

Reply:
It's not defined as the standard behavior, but the Payment Handler API is only available in Chrome currently anyway.